config/targets.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-env node */
+
+module.exports = {
+  browsers: [
+    'last 2 Chrome versions',
+    'last 2 Firefox versions',
+    'last 1 Safari versions',
+    'last 2 Edge versions'
+  ]
+};
```

Comment:
Is this our true support matrix? I can’t remember if we’ve ever gotten a clear answer on this…